v2/timeutils/milli_test.go

```diff
@@ -8,13 +8,13 @@ import (
 )
 
 func Test_MillisecondsShouldBeWithinRealisticInterval(t *testing.T) {
-	assert.True(t, MillisecondsNow() > 1528030261000) // 2018-03-06
-	assert.True(t, MillisecondsNow() < 9999999999999) // 2286-11-20
+	assert.True(t, MillisecondsNow() > 1528030261000) // nolint: gomnd
```

Comment:
This number I actually don't know.
And the other one I think should maxInt?

Reply:
I took them as magic numbers used in a test.

Reply:
But you removed the translation of the numbers? in the old test It was easy to see what dates we were actually comparing to. Now I need to translate the number to find out what realistic Intervall means.